components/search_results_item/search_results_item.jsx

```diff
@@ -85,24 +122,24 @@ export default class SearchResultsItem extends React.Component {
         Utils.updateWindowDimensions(this);
```

Comment:
`onUpdateWindowDimensions` might not need here.

Reply:
This will be changed with this PR: https://github.com/mattermost/mattermost-webapp/pull/435